File: FMAplication/Domain/Audit/AuditPOSMProduct.cs
using FMAplication.Core;
using FMAplication.Domain.Products;
using FMAplication.Enumerations;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace FMAplication.Domain.Audit
{
    public class AuditPOSMProduct:AuditableEntity<int>
    {
        public int AuditSetupId { get; set; }
        public AuditSetup AuditSetup { get; set; }
        public int POSMProductId { get; set; }
        public POSMProduct POSMProduct { get; set; }
        public ActionType ActionType { get; set; }

    }
}

File: FMAplication/ClientApp/src/app/Shared/Entity/Daily-Audit/auditPOSMProduct.ts
import { PosmProduct } from "..";
import { ActionType } from "../../Enums/actionType";

export class AuditPOSMProduct {
    id: number;
    auditSetupId: number;
    posmProductId: number;
    posmProduct: PosmProduct;
    actionType: ActionType;
}
